deny.toml

```diff
@@ -198,6 +198,10 @@ reason = "needed for test tracing.  Remove this exception when https://github.co
 name = "regex-syntax"
 reason = "needed for test tracing.  Remove this exception when https://github.com/tokio-rs/tracing/pull/3033 lands"
 
+[[bans.skip]]
+name = "syn"
+reason = "syn is basically always a build time dependency so we aren't shipping multiple versions"
```

Comment:
Is this necessary? It doesn't seem to be the case on `main`, but maybe you add a new version with your PR?

Reply:
I will confirm.  This commit is what confused me regarding the fate of #211 and which resulted in #235

I swear I had cargo deny passing in both PRs but in the morning it was a completely different issue so I don't know what happened